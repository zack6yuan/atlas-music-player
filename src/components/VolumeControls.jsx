import volume from "../assets/volume.svg";

export default function VolumeControls() {
    return (
        <div className="volume-controls">
            <img src={ volume } alt="" />
            <input type="range" min="0" max="100"/>
        </div>
    )
}