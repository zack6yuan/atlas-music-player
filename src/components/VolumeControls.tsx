import volume from "../assets/volume.svg";

export default function VolumeControls() {
    return (
        <div className="volume-controls flex">
            <img src={ volume } alt="volume-icon" className="mx-3 "/>
            <input type="range" min="0" max="100" className="accent-crimson 2xl:w-90 xl:w-96 lg:w-90 md:w-67 sm:w-122"/>
        </div>
    )
}