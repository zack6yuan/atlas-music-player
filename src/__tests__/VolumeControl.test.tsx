import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import VolumeControls from "../components/VolumeControls"

test("Volume Controls render correctly", () => {
    const { container } = render(
        <VolumeControls />
    );
    expect(container).toMatchSnapshot();
});