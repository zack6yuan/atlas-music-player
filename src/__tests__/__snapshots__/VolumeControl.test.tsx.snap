// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Volume Controls render correctly 1`] = `
<div>
  <div
    class="volume-controls flex"
  >
    <img
      alt="volume-icon"
      class="mx-3 "
      src="data:image/svg+xml,%3csvg%20width='24'%20height='24'%20viewBox='0%200%2024%2024'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cg%20clip-path='url(%23clip0_263_122)'%3e%3cpath%20d='M11%204.702C10.9998%204.56271%2010.9583%204.42661%2010.8809%204.31086C10.8034%204.19511%2010.6934%204.1049%2010.5647%204.05163C10.436%203.99835%2010.2944%203.98439%2010.1577%204.01151C10.0211%204.03863%209.89559%204.10561%209.797%204.204L6.413%207.587C6.2824%207.71838%206.12703%207.82253%205.95589%207.89343C5.78475%207.96432%205.60124%208.00054%205.416%208H3C2.73478%208%202.48043%208.10536%202.29289%208.29289C2.10536%208.48043%202%208.73479%202%209V15C2%2015.2652%202.10536%2015.5196%202.29289%2015.7071C2.48043%2015.8946%202.73478%2016%203%2016H5.416C5.60124%2015.9995%205.78475%2016.0357%205.95589%2016.1066C6.12703%2016.1775%206.2824%2016.2816%206.413%2016.413L9.796%2019.797C9.8946%2019.8958%2010.0203%2019.9631%2010.1572%2019.9904C10.2941%2020.0177%2010.436%2020.0037%2010.5649%2019.9503C10.6939%2019.8968%2010.804%2019.8063%2010.8815%2019.6902C10.959%2019.5741%2011.0002%2019.4376%2011%2019.298V4.702Z'%20fill='black'%20stroke='black'%20stroke-width='2'%20stroke-linecap='round'%20stroke-linejoin='round'/%3e%3cpath%20d='M16%209C16.6491%209.86548%2017%2010.9181%2017%2012C17%2013.0819%2016.6491%2014.1345%2016%2015'%20fill='black'/%3e%3cpath%20d='M16%209C16.6491%209.86548%2017%2010.9181%2017%2012C17%2013.0819%2016.6491%2014.1345%2016%2015'%20stroke='black'%20stroke-width='2'%20stroke-linecap='round'%20stroke-linejoin='round'/%3e%3cpath%20d='M19.364%2018.364C20.1998%2017.5283%2020.8627%2016.5361%2021.315%2015.4442C21.7673%2014.3522%2022.0001%2013.1819%2022.0001%2012C22.0001%2010.8181%2021.7673%209.64776%2021.315%208.55582C20.8627%207.46389%2020.1998%206.47173%2019.364%205.636'%20fill='black'/%3e%3cpath%20d='M19.364%2018.364C20.1998%2017.5283%2020.8627%2016.5361%2021.315%2015.4442C21.7673%2014.3522%2022.0001%2013.1819%2022.0001%2012C22.0001%2010.8181%2021.7673%209.64776%2021.315%208.55582C20.8627%207.46389%2020.1998%206.47173%2019.364%205.636'%20stroke='black'%20stroke-width='2'%20stroke-linecap='round'%20stroke-linejoin='round'/%3e%3c/g%3e%3cdefs%3e%3cclipPath%20id='clip0_263_122'%3e%3crect%20width='24'%20height='24'%20fill='white'/%3e%3c/clipPath%3e%3c/defs%3e%3c/svg%3e"
    />
    <input
      class="accent-crimson 2xl:w-90 xl:w-96 lg:w-90 md:w-67 sm:w-122"
      max="100"
      min="0"
      type="range"
    />
  </div>
</div>
`;
